### DISCUSSION TITLE
Supported Models of  iMac  with opencore legacy

### DISCUSSION THREAD

**Discussion by mrbill412** [community/community#139144] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Supported Models of  iMac  with opencore legacy patcher for any of these iMac models?
14,2?
13,1?
8,1?

**Comment by LiteBrite82** [comment]
Hi @mrbill412 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
